dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/model/AnalyticsTable.java

```diff
@@ -62,6 +63,8 @@ public class AnalyticsTable extends Table {
   /** Tracked entity type of enrollments in analytics table. */
   private TrackedEntityType trackedEntityType;
 
+  @Getter @Setter private boolean tableDistributed = false;
```

Comment:
Can we use `distributed` as the column name, as "table" is implicit from the class name `AnalyticsTable`?

Reply:
It has been renamed 👍